src/bindgen/ir/structure.rs

```diff
@@ -230,6 +255,33 @@ impl Source for Struct {
     }
 }
 
+fn format_function_call_1<W: Write>(f: &Function, out: &mut SourceWriter<W>) {
+    out.write("return ::");
```

Comment:
It'd be nice to omit the `return` if the return type is `void`, but from my brief testing it isn't a compile warning/error.